src/components/Player.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="box">
+  <div class="box" v-if="data && data.role">
```

Comment:
only show the panel if data exists. this prevents the template from throwing errors when we have just loaded the page